config/src/config/mempool_config.rs

```diff
@@ -96,10 +92,6 @@ pub struct MempoolConfig {
     pub num_sender_buckets: u8,
     /// Load balancing configuration for the mempool. This is used only by PFNs.
     pub load_balancing_thresholds: Vec<LoadBalancingThresholdConfig>,
-    /// When the load is low, PFNs send all the mempool traffic to only one upstream FN. When the load increases suddenly, PFNs will take
-    /// up to 10 minutes (shared_mempool_priority_update_interval_secs) to enable the load balancing. If this flag is enabled,
-    /// then the PFNs will always do load balancing irrespective of the load.
-    pub enable_max_load_balancing_at_any_load: bool,
```

Comment:
If this config is removed, we need to modify the corresponding code in priority.rs as well to remove the usage of it.

Reply:
It could take up to 10 minutes for load balancing to kick in. This setting could be helpful for the Aptos Experience demo. Are we sure we want to remove this?

Reply:
I just moved it from this flag into the first entry in the load_balancing_thresholds list, to simplify things out